src/components/ResetPasswordSuccess/ResetPasswordSuccess.css

```diff
@@ -0,0 +1,20 @@
+.reset-password-success {
+	border: var(--border);
+	border-radius: var(--rounded-md);
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-md) 0;
+	justify-content: center;
+	margin-top: var(--spacing-lg);
+	padding: var(--spacing-md);
+	& p {
+		font-size: var(--font-size-md);
+		text-align: center;
+	}
+}
+.Log-in-account-link {
```

Comment:
to keep the class names consistent, please refrain from using capitalization.
Suggested change:
`.login-account-link {`

Reply:
@bcha92 when sending a review there is a tiny button on the edit bar for suggesting changes:
<img width="58" height="53" alt="image" src="https://github.com/user-attachments/assets/b7a47adf-724f-4afc-91a7-05a7c768a50e" />

You can also use the `suggestion` tag on a fenced code block:
```
```suggestion
...
```